vehicle/src/main/scala/com/wedriveu/vehicle/control/VehicleBehavioursImpl.scala

```diff
@@ -102,6 +110,20 @@ class VehicleBehavioursImpl(selfDrivingVehicle: SelfDrivingVehicle) extends Vehi
       drainBattery()
       timePassedForBattery += timeStepBatteryConsumed
     }
+    stopUi.writeMessageLog("New Position = " + latInter + " , " + lonInter)
+  }
+
+  private def checkVehicleIsNotBroken(): Boolean = {
+    if (selfDrivingVehicle.state.equals("broken")) {
```

Comment:
So, why the "available", "broken" etc. status are not in a sort of "shared" constants?